mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -70,24 +71,10 @@
  * 
  */
 public abstract class MailboxManagerTest {
```

Comment:
this commit is impossible to review. Did you just moved things around or there's more ?

Reply:
Just moved things around